### ISSUE TITLE
Don’t Bow to Authority

### ISSUE THREAD

**Issue by JattMannu** [JattMannu/react-blog-github#17]
## Authority Bias

The first book of the Bible explains what happens when we disobey a great authority: We get ejected from paradise. This is also what less celestial authorities would have us believe—political pundits, scientists, doctors, CEOs, economists, government heads, sports commentators, consultants, and stock market gurus.

Authorities pose two main problems to clear thinking: First, their track records are often sobering. There are about one million trained economists on the planet, and not one of them could accurately predict the timing of the 2008 financial crisis (with the exception of Nouriel Roubini and Nassim Taleb), let alone how the collapse would play out, from the real estate bubble bursting to credit default swaps collapsing, right through to the full-blown economic crunch. Never has a group of experts failed so spectacularly. The story from the medical world is much the same: Up until 1900 it was discernibly wiser for patients to avoid doctor’s visits; too often the “treatment” only worsened the illness, due to poor hygiene and folk practices such as bloodletting.

Psychologist Stanley Milgram demonstrated the authority bias most clearly in an experiment in 1961. His subjects were instructed to administer ever-increasing electrical shocks to a person sitting on the other side of a pane of glass. They were told to start with 15 volts, then 30 volts, 45 volts, and so on, until they reached the maximum—a lethal dose of 450 volts. In reality, no electrical current was actually flowing; Milgram used an actor to play the role of the victim, but those charged with administering the shocks didn’t know that. The results were, well, shocking: As the person in the other room wailed and writhed in pain, and the subject administering the shock wanted to stop, the professor would say, “Keep going, the experiment depends on it.” The majority of people continued with the electrocution. More than half of the participants went all the way up to the maximum voltage—out of sheer obedience to authority.

Over the past decade, airlines have also learned the dangers of the authority bias. In the old days, the captain was king. His commands were not to be doubted. If a copilot suspected an oversight, he wouldn’t have dared to address it out of respect for—or fear of—his captain. Since this behavior was discovered, nearly every airline has instituted crew resource management (CRM), which coaches pilots and their crews to discuss any reservations they have openly and quickly. In other words: They carefully deprogram the authority bias. CRM has contributed more to flight safety in the past twenty years than have any technical advances.

Many companies are light-years from this sort of foresight. Especially at risk are firms with domineering CEOs, where employees are likely to keep their “lesser” opinions to themselves—much to the detriment of the business.

Authorities crave recognition and constantly find ways to reinforce their status. Doctors and researchers sport white coats. Bank directors don suits and ties. Kings wear crowns. Members of the military wield rank badges. Today, even more symbols and props are used to signal expertise: appearances on talk shows and on the covers of magazines, to book tours and Wikipedia entries. Authority changes much like fashion does, and society follows it just as much.

In conclusion: Whenever you are about to make a decision, think about which authority figures might be exerting an influence on your reasoning. And when you encounter one in the flesh, do your best to challenge him or her.
